impeller/typographer/backends/skia/text_frame_skia.cc

```diff
@@ -57,40 +58,29 @@ std::shared_ptr<TextFrame> MakeTextFrameFromTextBlobSkia(
   for (SkTextBlobRunIterator run(blob.get()); !run.done(); run.next()) {
     // TODO(jonahwilliams): ask Skia for a public API to look this up.
     // https://github.com/flutter/flutter/issues/112005
+
     SkStrikeSpec strikeSpec = SkStrikeSpec::MakeWithNoDevice(run.font());
     SkBulkGlyphMetricsAndPaths paths{strikeSpec};
 
     const auto glyph_count = run.glyphCount();
     const auto* glyphs = run.glyphs();
     switch (run.positioning()) {
       case SkTextBlobRunIterator::kFull_Positioning: {
-        std::vector<SkRect> glyph_bounds;
```

Comment:
These bounds are wrong, this computation is moved into the typographer contents where the exact scale is applied to the font size.